### ISSUE TITLE
Is this functional?

### ISSUE THREAD

**Issue by BrianGilbert** [haydnnewport/esphome_adapter#1]
@haydnnewport given the lack of documentation, just checking if this is functional?

**Comment by haydnnewport** [comment]
No, this is not yet functional, sorry - still a pet project I hope to get some time to work on.

Out of interest, what kind of esphome device were you looking to use it with?

**Comment by BrianGilbert** [comment]
@haydnnewport Sonoff devices that have been flashed with ESPHome, particularly S20 and S26 at this stage

**Comment by BrianGilbert** [comment]
@haydnnewport just posted this on webthings issue queue https://github.com/mozilla-iot/gateway/issues/2340

**Comment by BrianGilbert** [comment]
@haydnnewport when it is working, you just submit a PR here to have it added to WebThings: https://github.com/mozilla-iot/addon-list/tree/master/addons

**Comment by tim-hellhake** [comment]
@haydnnewport
I also wrote an adapter for the ESPHome firmware:
https://github.com/tim-hellhake/esphome-adapter
Maybe it saves you some time 😄
